### PR TITLE
ADEN-12189 Add LI to N&S

### PR DESCRIPTION
**Pull request by szymon-nawrocki**
### Ticket
https://fandom.atlassian.net/browse/ADEN-12189

### CODE REVIEW

File: src/ad-services/live-connect/index.ts [hunk 1]
```diff
@@ -89,9 +100,48 @@ class LiveConnect extends BaseServiceSetup {
 			return;
 		}
 		idConfigMapping.forEach(({ id, name, params }) => {
-			this.resolveAndReportId(id, name, params);
+			const storageKey = this.getConfigName(id);
```
Comment (domjanik) [inline src/ad-services/live-connect/index.ts:103]: getConfigName returns the same name as you get 1 line above in forEach. Is it really necessary to use this method here ?

**Review by domjanik — APPROVED** [review]

File: src/ad-services/live-connect/index.ts [hunk 2]
```diff
@@ -89,9 +100,48 @@ class LiveConnect extends BaseServiceSetup {
...
+		if (!this.storageConfig) {
+			return true;
+		}
+
+		let shouldLoadScript = false;
+
+		this.storageConfig.mandatoryParams.forEach((param) => {
+			const storageKey = this.getConfigName(param);
+			if (!this.isAvailableInStorage(storageKey)) {
+				shouldLoadScript = true;
```
Comment (mfarul) [inline src/ad-services/live-connect/index.ts:131]: and break ;)

File: src/ad-services/live-connect/index.ts (hunk 1 above)
Comment (szymon-nawrocki) [reply src/ad-services/live-connect/index.ts:103]: Good catch! It is not necessary. Fixed ✅

File: src/ad-services/live-connect/index.ts (hunk 2 above)
Comment (szymon-nawrocki) [reply src/ad-services/live-connect/index.ts]: done ✅

**Pull request merged**